lib/pbench/server/api/resources/query_apis/controllers_list.py

```diff
@@ -134,6 +126,9 @@ def postprocess(self, es_json: JSON, context: CONTEXT) -> JSON:
         ]
         """
         controllers = []
+        if context.get("NODATA"):
+            return jsonify(controllers)
```

Comment:
Referencing `controllers` at line 130 seems slightly obfuscating -- why not just return `jsonify([])` (similar to what you did in `lib/pbench/server/api/resources/query_apis/datasets_detail.py` and the other modules)?

(And, if you buy into that, you can do it before set up stuff, like defining `controllers`....)

Reply:
I nearly did that, but in this case I had a clean point to return with a known local `{}` already and decided to use it. I'm not quite sure why I went with that knowing you'd object ... maybe I was just feeling contrary that day. 😆